fmlonly/src/main/resources/coremods/load_fml_server.js

```diff
@@ -0,0 +1,35 @@
+var ASMAPI = Java.type('net.minecraftforge.coremod.api.ASMAPI')
+var Opcodes = Java.type('org.objectweb.asm.Opcodes')
+
+function initializeCoreMod() {
+    return {
+        'minecraft': {
+            'target': {
+                'type': 'METHOD',
+                'class': 'net.minecraft.server.Main',
+                'methodName': 'main',
+                'methodDesc': '([Ljava/lang/String;)V'
+            },
+            'transformer': function(methodNode) {
+                var meth = ASMAPI.getMethodNode();
+                meth.visitMethodInsn(Opcodes.INVOKESTATIC, 'net/minecraftforge/fmlonlyserver/ServerModLoader', 'load', '()V', false)
+
+                var methOptions = ASMAPI.getMethodNode();
+                methOptions.visitIntInsn(Opcodes.ALOAD, 1);
+                methOptions.visitMethodInsn(Opcodes.INVOKESTATIC, 'net/minecraftforge/fmlonlyserver/ServerModLoader', 'addOptions', '(Ljoptsimple/OptionParser;)V', false)
+
+                for (var i=0; i<methodNode.instructions.size(); i++) {
+                    var ain = methodNode.instructions.get(i);
+                    if (ain.getOpcode() == Opcodes.INVOKESPECIAL && ain.owner == 'joptsimple/OptionParser' && ain.name == '<init>' && ain.desc == '()V') {
+                        // Inject after the STORE
+                        methodNode.instructions.insert(methodNode.instructions.get(i + 1), methOptions.instructions);
+                    } else if (ain.getOpcode() == Opcodes.INVOKESTATIC && ain.owner == 'net/minecraft/Util' && ain.name == ASMAPI.mapMethod('m_137584_') && ain.desc == '()V') {
```

Comment:
Please add the mapped name of the method (and the version of that mapped name too) as a comment, for future quick reference. Do the same for the `load_fml_client.js` coremod as well.